source/guides/maintenance/fix-broken-install.rst

```diff
@@ -43,21 +43,22 @@ Mount root partition, verify, and fix
 
 #. Ensure the system is connected to the network.
 
-#. Mount the system’s root partition.
+#. Mount the system’s root and boot partitions.
 
-   #. To find the root partition, run:
+   #. To find the root and boot partitions, run:
 
       .. code-block:: bash
 
-         lsblk
+         lsblk -o NAME,LABEL,PARTTYPE,PARTLABEL
```

Comment:
Good suggestion.

Reply:
I agree. This is helpful in viewing the output. In summary, please delete all other changes on a new commit to this PR, and then we'll accept for merge. Cheers, @mns6070 .